.vscode/extensions.json

```diff
@@ -1,6 +1,7 @@
 {
   "recommendations": [
     "biomejs.biome",
-    "dbaeumer.vscode-eslint"
+    "dbaeumer.vscode-eslint",
+    "ms-vscode.azure-account"
```

Comment:
Is there any specific reason on why we are recommending this extension into the repo? This is handled by the extension, therefore I think not having it should be fine